### DISCUSSION TITLE
Is it possible to get the user activity in github cloud enterprise ?

### DISCUSSION THREAD

**Discussion by dhikrahashim** [community/community#68314] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

Hi,

We want to check our users code  activity in our github cloud enterprise on each organization.

**Comment by HassanSj** [comment]
As of my last knowledge update in September 2021, GitHub Enterprise Cloud does not provide built-in features for tracking or monitoring user activity at a granular level. GitHub Enterprise Cloud primarily focuses on providing a secure and collaborative environment for software development. While it offers audit logs and security features, these are often related to repository and access controls rather than detailed user activity monitoring.

**Reply by dhikrahashim** [reply]
https://docs.github.com/en/organizations/managing-membership-in-your-organization/exporting-member-information-for-your-organization

**Comment by ghostinhershell** [comment]
Hi @dhikrahashim and @HassanSj, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
